app/js/conceptLocale.service.js

```diff
@@ -0,0 +1,78 @@
+(function() {
+    'use strict';
+    
+	angular
+		.module('conceptDictionaryApp')
+		.factory('conceptLocaleService', conceptLocaleService);
+				
+
+		function conceptLocaleService(){
+					return{
+						getLocales: getLocales,
+						getLocaleNames: getLocaleNames,
+						getLocaleDescr: getLocaleDescr
+					}
+		};
+		/**
+		 * @descriptions array of descriptions objects of concept
+		 * @names array of names objects of concept
+		 * @serverLocales array of available locales
+		 * @returns array of locales of names and descriptions
+		 */
+		function getLocales (names, descriptions, serverLocales){
```

Comment:
functions belonging to conceptLocaleService should be declared within conceptLocalService above line 15.